### ISSUE TITLE
Repeated prompt when Input task is used within Then block (Trac #1006)

### ISSUE THREAD

**Issue by phing-issues-importer** [phingofficial/phing#557]
Using an input task within the `then` block of an `if` statement causes the text or message to be printed four times instead of one.

Some exploration indicates that the addText method is being called four times, so it appends the text to itself.

---

build.xml:

``` text
<?xml version="1.0"?>
<project name="test" default="test">

    <target name="test">
        <if>
            <equals arg1="true" arg2="true" />
            <then>
                <input propertyname="test" defaultValue="no" validargs="yes,no">Phing?</input>
            </then>
        </if>
    </target>

</project>
```

Expected:

``` text
Phing?(yes/no) [no]
```

Actual:

``` text
Phing?Phing?Phing?Phing?(yes/no) [no]
```

Migrated from https://www.phing.info/trac/ticket/1006

``` json
{
    "status": "new", 
    "changetime": "2013-06-14T23:43:47", 
    "description": "Using an input task within the `then` block of an `if` statement causes the text or message to be printed four times instead of one.\n\nSome exploration indicates that the addText method is being called four times, so it appends the text to itself.\n\n----\nbuild.xml:\n{{{\n<?xml version=\"1.0\"?>\n<project name=\"test\" default=\"test\">\n\n\t<target name=\"test\">\n\t\t<if>\n\t\t\t<equals arg1=\"true\" arg2=\"true\" />\n\t\t\t<then>\n\t\t\t\t<input propertyname=\"test\" defaultValue=\"no\" validargs=\"yes,no\">Phing?</input>\n\t\t\t</then>\n\t\t</if>\n\t</target>\n\n</project>\n}}}\n\nExpected:\n{{{\nPhing?(yes/no) [no]\n}}}\n\nActual:\n{{{\nPhing?Phing?Phing?Phing?(yes/no) [no]\n}}}", 
    "reporter": "doug.fitzmaurice@ents24.com", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1371253427358917", 
    "component": "phing-core", 
    "summary": "Repeated prompt when Input task is used within Then block", 
    "priority": "tbd", 
    "keywords": "", 
    "version": "2.5.0", 
    "time": "2013-04-19T12:42:56", 
    "milestone": "", 
    "owner": "mrook", 
    "type": "defect"
}
```

**Comment by phing-issues-importer** [comment]
Trac update at 20130614T23:43:47: wesdeboer@gmail.com commented:

> I'm having the same problem, I tracked it down further to commit: https://github.com/phingofficial/phing/commit/36e647208098fe00530564a6c41cd9224f23d199
> 
> Re-add the line back in and it solves the problem

**Comment by mrook** [comment]
This was actually fixed in c496443b3e38c9ffd98059b6ecd1743b0db63c50.
